alembic/versions/1c87fd8da02e_adds_job_visibility_column.py

```diff
@@ -0,0 +1,142 @@
+#     else:
+#         op.alter_column('table_name', 'column_name', type_=sa.Unicode(), server_default='', nullable=False)
+#
+# ===========================================================================
+
+
+dept_membership_request_table = table(
+    'dept_membership_request',
+    sa.Column('id', sideboard.lib.sa.UUID()),
+    sa.Column('attendee_id', sideboard.lib.sa.UUID()),
+    sa.Column('department_id', sideboard.lib.sa.UUID()),
+)
+
+
+def upgrade():
+    if is_sqlite:
+        with op.batch_alter_table('job', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.add_column(sa.Column('visibility', sa.Integer(), server_default='0', nullable=False))
+            batch_op.create_index('ix_job_department_id', 'job', ['department_id'], unique=False)
+
+        with op.batch_alter_table('dept_membership', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_dept_membership_attendee_id', 'dept_membership', ['attendee_id'], unique=False)
+            batch_op.create_index('ix_dept_membership_department_id', 'dept_membership', ['department_id'], unique=False)
+
+        with op.batch_alter_table('dept_membership_dept_role', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_dept_membership_dept_role_dept_membership_id', 'dept_membership_dept_role', ['dept_membership_id'], unique=False)
+            batch_op.create_index('ix_dept_membership_dept_role_dept_role_id', 'dept_membership_dept_role', ['dept_role_id'], unique=False)
+
+        with op.batch_alter_table('dept_membership_request', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_dept_membership_request_attendee_id', 'dept_membership_request', ['attendee_id'], unique=False)
+            batch_op.create_index('ix_dept_membership_request_department_id', 'dept_membership_request', ['department_id'], unique=False)
+
+        with op.batch_alter_table('dept_role', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_dept_role_department_id', 'dept_role', ['department_id'], unique=False)
+
+        with op.batch_alter_table('job_required_role', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_job_required_role_dept_role_id', 'job_required_role', ['dept_role_id'], unique=False)
+            batch_op.create_index('ix_job_required_role_job_id', 'job_required_role', ['job_id'], unique=False)
+
+        with op.batch_alter_table('shift', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.create_index('ix_shift_attendee_id', 'shift', ['attendee_id'], unique=False)
+            batch_op.create_index('ix_shift_job_id', 'shift', ['job_id'], unique=False)
+    else:
+        op.add_column('job', sa.Column('visibility', sa.Integer(), server_default='0', nullable=False))
+
+        op.create_index('ix_job_department_id', 'job', ['department_id'], unique=False)
```

Comment:
Not all of these indexes are strictly necessary, but they're all on low-write-volume tables, so they won't hurt anything. They should make a lot of queries a little faster in some places, and a few queries much faster.